relayer/processor/message_processor.go

```diff
@@ -330,6 +330,10 @@ func (mp *messageProcessor) trackAndSendMessages(
 	needsClientUpdate bool,
 ) error {
 	broadcastBatch := dst.chainProvider.ProviderConfig().BroadcastMode() == provider.BroadcastModeBatch
+	maxMsgNum := dst.chainProvider.ProviderConfig().BroadcastMaxMsgNum()
+	if maxMsgNum < 2 && maxMsgNum != 0 {
+		maxMsgNum = 2
+	}
```

Comment:
is there any case where it would even make sense to set this variable to a value less than 2? i'm guessing it would always need to be at least 2 since we are attempting to broadcast some msg along with the `MsgUpdateClient`

Reply:
Ahh right this check is to ensure the user chosen value is always at least 2. 

Could this check then be simplified to:

```go
if maxMsgNum < 2 {
   maxMsgNum = 2
}
```

Since the `maxMsgNum < 2` would also account for the case of `maxMsgNum == 0` since `0 < 2`.

Perhaps I'm overlooking something here.

I would be okay with at least adding a log inside of the `if statement` when we auto correct the `maxMsgNum` value to 2, that way the user at least is away that they have configured an invalid value.